CMakeLists.txt

```diff
@@ -193,6 +193,18 @@ if (NOT HIPIFY_CLANG_TESTS_ONLY)
     PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/bin/hipexamine-perl.sh
     DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/hipify)
 
+  # Adding symlink for hipify-perl for backward compatibility
+  # Note: To be removed once hipify-perl install location usages
+  # are fully updated across modules Or hipify-perl is deprecated.
+  install(
+    CODE "execute_process(
+        COMMAND bash -c \"set -e
```

Comment:
Unacceptable usage of *nix command unconditionally, as at least the Windows target is broken.